spec/asyncapi.md

```diff
@@ -2627,6 +2627,31 @@ Message Payload Property | `$message.payload#/messageId` | Correlation ID is set
 
 Runtime expressions preserve the type of the referenced value.
 
+### <a name="traitsMergeMechanism"></a>Traits Merge Mechanism
+
+Traits MUST be merged with the main object using the [JSON Merge Patch](https://tools.ietf.org/html/rfc7386) algorithm in the same order they are defined. A property on a trait MUST NOT override the same property on the main object.
```

Comment:
I would be more explicit on the exception over JSON Merge Patch mechanism on not overriding properties on the target. Like mentioning it is **literally an exception**

Reply:
Additionally, I will suggest using wording from [JSON Merge Patch RFC](https://tools.ietf.org/html/rfc7386), for example calling "target" to what you called "main object"

Reply:
Target is good, but you need to keep in mind that the whole problem that JSON Merge patch addresses and describes is an inverse situation than merging traits ;)

Reply:
So what to call it?

Reply:
Solved